pkgs/servers/web-apps/slskd/default.nix

```diff
@@ -0,0 +1,71 @@
+{ lib, stdenv
+, fetchFromGitHub
+, fetchurl
+, unzip
+, dotnetCorePackages
+, buildDotnetModule
+, mono
+}:
+let
+  pname = "slskd";
+  version = "0.17.5";
+
+  # TODO build this via npm
+  wwwroot = stdenv.mkDerivation {
+    inherit version;
+    pname = "${pname}-wwwroot";
+
+    src = fetchurl {
+      url = "https://github.com/slskd/slskd/releases/download/${version}/slskd-${version}-linux-x64.zip";
+      sha256 = "sha256-QYGu2kwZJPMXeSYcE2HpC44RvSOl4MUgNp1Z+gTvrck=";
+    };
+
+    nativeBuildInputs = [ unzip ];
+
+    unpackPhase = ''
+      unzip -q $src
+    '';
+
+    sourceRoot = "./wwwroot";
+
+    installPhase = ''
+      cp -r . $out
+    '';
+  };
+
+in buildDotnetModule {
+  inherit pname version;
+
+  src = fetchFromGitHub {
+    owner = "slskd";
+    repo = "slskd";
+    rev = version;
+    sha256 = "sha256-iIM29ZI3M9etbw4yzin+4f4cGHIt5qjIl7uzsTUCBc4=";
+  };
+
+  runtimeDeps = [ mono ];
+
+  dotnet-sdk = dotnetCorePackages.sdk_7_0;
+  dotnet-runtime = dotnetCorePackages.aspnetcore_7_0;
+
+  projectFile = "slskd.sln";
```

Comment:
I've never used .net, but what's the difference between pointing the project file to slskd.sln or to src/slskd/slskd.csproj?

Reply:
No idea! I just did it that way following `buildDotnetModule`'s doc and it worked haha! First experience with .NET for me.